owner/fail_test.go

```diff
@@ -63,7 +57,27 @@ var (
 )
 
 func (s *testSuite) TestFailNewSession(c *C) {
-	defer testleak.AfterTest(c)()
+	ln, err := net.Listen("unix", "new_session:12379")
+	c.Assert(err, IsNil)
+	srv := grpc.NewServer(grpc.ConnectionTimeout(time.Minute))
+	var stop sync.WaitGroup
+	stop.Add(1)
+	go func() {
+		if err = srv.Serve(ln); err != nil {
```

Comment:
Why do we call this function in this test?

Reply:
because this test will be failed after upgrade grpc due can not establish connection to `net.Listen("unix", "new_session:12379")`,  old grpc can be ready before handshake success, but in new grpc this changed(https://github.com/grpc/grpc-go/issues/2406), so we need setup a grpc serve for listened port